src/search.ts

```diff
@@ -32,8 +34,8 @@ export interface Position {
 export async function search(content: string): Promise<SearchValue> {
   const tree = await acorn.parse(content, {locations: true});
   const nodeArr: Node[] = getRequireCalls(tree);
-  const result: SearchValue = getRequiredModules(nodeArr);
-  
+  const result: SearchValue = getRequiredModules(nodeArr, iOModules);
```

Comment:
Instead of passing it here, I would suggest filtering the results list based on `ioModules`.

Reply:
done